### PR TITLE
Updated github actions.

### PR DESCRIPTION
**Pull request by lauri-codes**

### CODE REVIEW

**Comment by lauri-codes** [comment]
@ladinesa: Could you review and merge this? I just updated the repo to use latest github actions, project setup etc. to fix the CI problems.

**Review by ladinesa — DISMISSED** [review]

File: eelsdbparser/crawl.py [hunk 1]
```diff
@@ -25,24 +25,21 @@
...
-            data = requests.get(entry['download_link']).content
-            with zf.open(os.path.join(path, 'data.msa'), mode='w') as f:
-                f.write(data)
-
-        except Exception:
-            print('Could not download spectra for %s' % entry['permalink'])
-
-    zf.close()
+    with zipfile.ZipFile('eels-data.zip', mode='x') as zf:
+        eelsdb = requests.get(f'https://api.eelsdb.eu/spectra?per_page={max}').json()
```
Comment (ladinesa) [inline eelsdbparser/crawl.py]: should we put a try here?

Comment (lauri-codes) [reply eelsdbparser/crawl.py]: Yes, I added it.

Comment (lauri-codes) [reply eelsdbparser/crawl.py]: Can you approve once you've taken a new look? Can't merge until approved.

**Review by ladinesa — APPROVED** [review]

**Pull request merged**